simulator/simple_planning_simulator/launch/simple_planning_simulator.launch.py

```diff
@@ -80,7 +80,24 @@ def launch_setup(context, *args, **kwargs):
         executable="static_transform_publisher",
         name="static_map_to_odom_tf_publisher",
         output="screen",
-        arguments=["0.0", "0.0", "0.0", "0", "0", "0", "map", "odom"],
+        arguments=[
+            "--frame-id",
+            "map",
+            "--child-frame-id",
+            "odom",
+            "--x",
+            "0.0",
+            "--y",
+            "0.0",
+            "--z",
+            "0.0",
+            "--roll",
+            "0.0",
+            "--pitch",
+            "0.0",
+            "--yaw",
+            "0.0",
+        ],
```

Comment:
All parameters are set to `0.0` by default (except for the `frame-id` and `child-frame-id` ) so you can remove them if you prefer.
```suggestion
        arguments=[
            "--frame-id",
            "map",
            "--child-frame-id",
            "odom",
        ],
```

Reply:
Thank you!